### PR TITLE
reducing log output(mainly the TRACE output from Consensus)

### PR DESCRIPTION
**Pull request by Arghya115**

### CODE REVIEW

File: src/Lachain.Consensus/AbstractProtocol.cs [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void Terminate()
             lock (_queueLock)
             {
                 if (Terminated) return;
-                Logger.LogTrace($"{Id}: protocol is terminated");
+            //    Logger.LogTrace($"{Id}: protocol is terminated");
```
Comment (Arghya115) [inline src/Lachain.Consensus/AbstractProtocol.cs:95]: 4.5%

File: src/Lachain.Consensus/AbstractProtocol.cs [hunk 2]
```diff
@@ -144,8 +144,8 @@ public void ReceiveMessage(MessageEnvelope message)
         {
             lock (_queueLock)
             {
-                if (Terminated)
-                    Logger.LogTrace($"{Id}: got message after termination");
+                if (Terminated){}
+            //        Logger.LogTrace($"{Id}: got message after termination");
```
Comment (Arghya115) [inline src/Lachain.Consensus/AbstractProtocol.cs:148]: 3%

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs [hunk 3]
```diff
@@ -59,15 +59,15 @@ private void TryProgressEpoch()
                     // epoch mod 2 = 0 -> we have not yet initiated BB
                     if (_currentEpoch != 0 && !_coins.ContainsKey(_currentEpoch - 1))
                     {
-                        Logger.LogTrace(
-                            $"{_agreementId}: can't progress epoch, blocked, coin (Ep={_currentEpoch - 1}) not present");
+                    /*    Logger.LogTrace(
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:62]: 0.5%

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs [hunk 4]
```diff
@@ -59,15 +59,15 @@ private void TryProgressEpoch()
...
+                            $"{_agreementId}: can't progress epoch, blocked, coin (Ep={_currentEpoch - 1}) not present"); */
                         return; // we cannot progress since coin is not tossed and estimate is not correct
                     }
 
-                    Logger.LogTrace(
+                /*    Logger.LogTrace(
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:67]: 2.12%

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs [hunk 5]
```diff
@@ -118,13 +118,13 @@ private void TryProgressEpoch()
                     // epoch mod 2 = 1 -> we have not yet tossed coin
                     if (!_binaryBroadcastsResults.ContainsKey(_currentEpoch - 1))
                     {
-                        Logger.LogTrace(
+                    /*    Logger.LogTrace(
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:121]: 2.14%

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs [hunk 6]
```diff
@@ -118,13 +118,13 @@ private void TryProgressEpoch()
...
                             $"{_agreementId}: can't progress epoch, blocked, BB (Ep={_currentEpoch - 1}) not present"
-                        );
+                        ); */
                         return; // we cannot progress since BB is not completed
                     }
 
-                    Logger.LogTrace($"{_agreementId}: epoch progressed, BB (Ep={_currentEpoch - 1}) is present");
+                //    Logger.LogTrace($"{_agreementId}: epoch progressed, BB (Ep={_currentEpoch - 1}) is present");
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:127]: 2.6%

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs [hunk 7]
```diff
@@ -221,9 +221,9 @@ private void RevisitConfMessages()
             if (_result != null) return;
             if (_binValues.Values().Sum(b => _receivedAux[b ? 1 : 0]) < N - F) return;
             _result = ChoseMinimalSet();
-            Logger.LogTrace($"{_broadcastId}: aux cnt = 0 -> {_receivedAux[0]}, 1 -> {_receivedAux[1]}");
-            Logger.LogTrace($"{_broadcastId}: current bin_values = {_binValues}");
-            Logger.LogTrace($"{_broadcastId}: and sum of aux on bin_values is {_binValues.Values().Sum(b => _receivedAux[b ? 1 : 0])}");
+        //    Logger.LogTrace($"{_broadcastId}: aux cnt = 0 -> {_receivedAux[0]}, 1 -> {_receivedAux[1]}");
+        //    Logger.LogTrace($"{_broadcastId}: current bin_values = {_binValues}");
+        //    Logger.LogTrace($"{_broadcastId}: and sum of aux on bin_values is {_binValues.Values().Sum(b => _receivedAux[b ? 1 : 0])}");
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:226]: 5.23% combining these 3 lines

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs [hunk 8]
```diff
@@ -232,7 +232,7 @@ private void RevisitAuxMessages()
         {
             if (_confSent) return;
             if (_binValues.Values().Sum(b => _receivedAux[b ? 1 : 0]) < N - F) return;
-            Logger.LogTrace($"{_broadcastId}: conf message sent with set {_binValues}");
+        //    Logger.LogTrace($"{_broadcastId}: conf message sent with set {_binValues}");
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:235]: 2%

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs [hunk 9]
```diff
@@ -288,7 +288,7 @@ private void CheckResult()
         {
             if (!_result.HasValue) return;
             if (_requested != ResultStatus.Requested) return;
-            Logger.LogTrace($"{_broadcastId}: made result {_result.Value.ToString()}");
+        //    Logger.LogTrace($"{_broadcastId}: made result {_result.Value.ToString()}");
```
Comment (Arghya115) [inline src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:291]: 2.27%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs [hunk 10]
```diff
@@ -208,7 +208,7 @@ public void Dispatch(ConsensusMessage message, int from)
                 _callback[request.To] = request.From;
             }
 
-            Logger.LogTrace($"Protocol {request.From} requested result from protocol {request.To}");
+        //    Logger.LogTrace($"Protocol {request.From} requested result from protocol {request.To}");
```
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:211]: 12.48%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs [hunk 11]
```diff
@@ -218,7 +218,7 @@ public void Dispatch(ConsensusMessage message, int from)
         public void InternalResponse<TId, TResultType>(ProtocolResult<TId, TResultType> result)
             where TId : IProtocolIdentifier
         {
-            Logger.LogTrace($"Protocol {result.From} returned result");
+        //    Logger.LogTrace($"Protocol {result.From} returned result");
```
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:221]: 4.1%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs [hunk 12]
```diff
@@ -277,7 +277,7 @@ public void Terminate()
         {
             ValidateId(id);
             if (_registry.TryGetValue(id, out var existingProtocol)) return existingProtocol;
-            Logger.LogTrace($"Creating protocol {id} on demand");
+        //    Logger.LogTrace($"Creating protocol {id} on demand");
```
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:280]: 4.43%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs [hunk 13]
```diff
@@ -234,12 +234,12 @@ public void Dispatch(ConsensusMessage message, int from)
...
                 {
                     cbProtocol?.ReceiveMessage(new MessageEnvelope(result, GetMyId()));
-                    Logger.LogTrace($"Result from protocol {result.From} delivered to {senderId}");
+                //    Logger.LogTrace($"Result from protocol {result.From} delivered to {senderId}");
                 }
             }
 
             // message is also delivered to self
-            Logger.LogTrace($"Result from protocol {result.From} delivered to itself");
+        //    Logger.LogTrace($"Result from protocol {result.From} delivered to itself");
```
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:242]: 4.03%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs (hunk 10 above)
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:211]: 4.42%

File: src/Lachain.Core/Consensus/EraBroadcaster.cs [hunk 14]
```diff
@@ -234,12 +234,12 @@ public void Dispatch(ConsensusMessage message, int from)
                 else
                 {
                     cbProtocol?.ReceiveMessage(new MessageEnvelope(result, GetMyId()));
-                    Logger.LogTrace($"Result from protocol {result.From} delivered to {senderId}");
+                //    Logger.LogTrace($"Result from protocol {result.From} delivered to {senderId}");
```
Comment (Arghya115) [inline src/Lachain.Core/Consensus/EraBroadcaster.cs:237]: 4.03%

File: src/Lachain.Networking/Hub/ClientWorker.cs [hunk 15]
```diff
@@ -99,14 +99,14 @@ private void Worker()
...
-                        $"Sending {toSend.Messages.Count} messages to hub, {megaBatchBytes.Length} bytes total, peer = {PeerPublicKey.ToHex()}");
+                /*    Logger.LogTrace(
+                        $"Sending {toSend.Messages.Count} messages to hub, {megaBatchBytes.Length} bytes total, peer = {PeerPublicKey.ToHex()}"); */
                     var messageTypes = toSend.Messages.Select(m =>
                         m.MessageCase != NetworkMessage.MessageOneofCase.ConsensusMessage
                             ? m.MessageCase.ToString()
                             : m.ConsensusMessage.PrettyTypeString()
                     );
-                    Logger.LogTrace($"Messages types: {string.Join("; ", messageTypes)}");
+                //    Logger.LogTrace($"Messages types: {string.Join("; ", messageTypes)}");
```
Comment (Arghya115) [inline src/Lachain.Networking/Hub/ClientWorker.cs:109]: 4.1%

File: src/Lachain.Consensus/AbstractProtocol.cs (hunk 1 above)
Comment (sgladkov) [reply src/Lachain.Consensus/AbstractProtocol.cs:95]: Better to have this record,  important for consensus state understanding

File: src/Lachain.Consensus/AbstractProtocol.cs (hunk 2 above)
Comment (sgladkov) [reply src/Lachain.Consensus/AbstractProtocol.cs:148]: Agree,  can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs (hunk 3 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:62]: Agree,  can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs (hunk 4 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:67]: Agree, can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs (hunk 5 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:121]: Agree,  can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs (hunk 6 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryAgreement.cs:127]: Agree,  can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs (hunk 7 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:226]: Agree,  can remove

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs (hunk 8 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:235]: Better to have this record

File: src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs (hunk 9 above)
Comment (sgladkov) [reply src/Lachain.Consensus/BinaryAgreement/BinaryBroadcast.cs:291]: Better to have this record

File: src/Lachain.Core/Consensus/EraBroadcaster.cs (hunk 10 above)
Comment (sgladkov) [reply src/Lachain.Core/Consensus/EraBroadcaster.cs:211]: Better to have this record

File: src/Lachain.Core/Consensus/EraBroadcaster.cs (hunk 11 above)
Comment (sgladkov) [reply src/Lachain.Core/Consensus/EraBroadcaster.cs:221]: Better to have this record

File: src/Lachain.Core/Consensus/EraBroadcaster.cs (hunk 14 above)
Comment (sgladkov) [reply src/Lachain.Core/Consensus/EraBroadcaster.cs:237]: Better to have this record

File: src/Lachain.Core/Consensus/EraBroadcaster.cs (hunk 12 above)
Comment (sgladkov) [reply src/Lachain.Core/Consensus/EraBroadcaster.cs:280]: Better to have this record

File: src/Lachain.Networking/Hub/ClientWorker.cs [hunk 16]
```diff
@@ -99,14 +99,14 @@ private void Worker()
                     var megaBatchBytes = megaBatch.ToByteArray();
                     if (megaBatchBytes.Length == 0)
                         throw new Exception("Cannot send empty message");
-                    Logger.LogTrace(
-                        $"Sending {toSend.Messages.Count} messages to hub, {megaBatchBytes.Length} bytes total, peer = {PeerPublicKey.ToHex()}");
+                /*    Logger.LogTrace(
```
[the remark this answers is not in the document]
Comment (sgladkov) [reply src/Lachain.Networking/Hub/ClientWorker.cs:102]: Better to have this record

File: src/Lachain.Networking/Hub/ClientWorker.cs (hunk 15 above)
Comment (sgladkov) [reply src/Lachain.Networking/Hub/ClientWorker.cs:109]: Better to have this record

**Review by sgladkov — CHANGES_REQUESTED** [review]
Return commented log records back please

**Comment by Arghya115** [comment]
> Return commented log records back please

returned, check please

**Review by sgladkov — APPROVED** [review]

**Pull request merged**